web/src/stream_popover.js

```diff
@@ -173,6 +173,7 @@ function build_stream_popover(opts) {
                 $(this).closest(".popover").fadeOut(500).delay(500).remove();
```

Comment:
Are these two lines something we should delete? I think the `.popover` class predates the Tippy migration, and I'm not sure what the `unsub` class does..